lib/core/enums/PublicKeyUsage.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Sidetree public key usage.
+ */
+enum PublicKeyUsage {
```

Comment:
Nit: can we rename to KeyUsage? Because it is not so much that it is usage of the "public key" itself. But not too insistent on it.